app/v1/services/AmendOtherReliefsService.scala

```diff
@@ -38,15 +37,13 @@ class AmendOtherReliefsService @Inject() (connector: AmendOtherReliefsConnector)
       logContext: EndpointLogContext,
       correlationId: String): Future[ServiceOutcome[Unit]] = {
 
-    val result = for {
-      responseWrapper <- EitherT(connector.amend(request)).leftMap(mapDownstreamErrors(desErrorMap))
-    } yield responseWrapper
+    val result = EitherT(connector.amend(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))
 
     result.value
   }
 
-  private def desErrorMap =
-    Map(
+  private def downstreamErrorMap: Map[String, MtdError] = {
+    val desErrors = Map(
       "INVALID_TAXABLE_ENTITY_ID"        -> NinoFormatError,
       "FORMAT_TAX_YEAR"                  -> TaxYearFormatError,
```

Comment:
I think `FORMAT_TAX_YEAR` is a mistake that has been cropping up in a few TYS endpoint PRs that i have seen and it should actually be `INVALID_TAX_YEAR`. Therefore, we can update line 48 and remove line 57. The same applies to in the service spec and ControllerISpec